code/modules/antagonists/changeling/powers/adrenaline.dm

```diff
@@ -19,6 +19,12 @@
 	user.SetKnockdown(0)
 	user.reagents.add_reagent("changelingadrenaline", 10)
 	user.reagents.add_reagent("changelinghaste", 2) //For a really quick burst of speed
-	user.adjustStaminaLoss(-75)
+	user.adjustStaminaLoss(-200)
+	user.updatehealth()
+	user.update_stamina()
+	user.stuttering = 0
+	user.resting = 0
+	user.lying = 0
```

Comment:
Ditto

Reply:
Will fix as above.

Reply:
After a lot of testing, I'm actually ready to defend the existence of those procs in the PR.
Without them, stamina updates the tick after, rather than immediately.